### ISSUE TITLE
Improve the default layout

### ISSUE THREAD

**Issue by goanpeca** [spyder-ide/spyder#2549]
Spyder should include the outline explorer pane by default on restart and perhaps it should go in the left, instead of the right, as many other IDEs are doing!

**Comment by Nodd** [comment]
:+1:

**Comment by jnsebgosselin** [comment]
I do not understand what needs to be done to solve this issue. @goanpeca or @ccordoba12 please explain and I'll try to implement it.

**Comment by ccordoba12** [comment]
@jnsebgosselin, this is what @goanpeca and me would like to see

![seleccion_016](https://user-images.githubusercontent.com/365293/46436423-64452700-c71e-11e8-9b21-17a3c71e8214.png)

`Documentos` (Documents in Spanish) roughly correspond to our Outline Explorer in `Proyectos` (Projects) to our Projects pane. We think this will give much more visibility to these plugins (especially the Outline Explorer) and it'll make us closer to other IDEs and also JupyterLab.

**Comment by jnsebgosselin** [comment]
@ccordoba12 hmm, interesting. In fact, this is exactly the setup I'm using.

So if I understand correctly, to fix this issue we need to make changes to the `Spyder Default Layer` in the `Windows Layer` to have this setup by default when opening Spyder for the first time? Or is there more that needs to be done?

![image](https://user-images.githubusercontent.com/10170372/46437472-74163880-c729-11e8-8a6d-27c406794cde.png)

**Comment by jnsebgosselin** [comment]
Or maybe what you are suggestion is to have a permanent vertical toolbar when a set of predefined panes can be selected, like in LibreOffice Writer?

![libre_office_vert_collection](https://user-images.githubusercontent.com/10170372/46438047-e3405c80-c72a-11e8-8eec-47fa23850ccf.gif)

**Comment by ccordoba12** [comment]
> Or maybe what you are suggesting is to have a permanent vertical toolbar where a set of predefined panes can be selected

Yes, that's exactly what I'm suggesting.

**Comment by jnsebgosselin** [comment]
Ok, I understand.

So if we implement that, would the `Outline Explorer` and `Project Explorer` still be available as dock widget ?

**Comment by ccordoba12** [comment]
No, but I think that's fine.

**Comment by jnsebgosselin** [comment]
Yes I agree with you

**Comment by ccordoba12** [comment]
Great! The thing is they make sense when shown next to the Editor (as with other IDEs).

**Comment by bcolsen** [comment]
## Thought one
Think file explorer with only a names column could work over there as well:
![image](https://user-images.githubusercontent.com/10513354/46515025-9ccc2a00-c81e-11e8-98ad-deccb82ddc75.png)

## Thought 2
I think the side tabs will leave a lot of wasted space for two or three tabs. There is only so much horizontal space in such a narrow section to show extended trees and long names. Three tabs at the top or bottom simply named (Projects, Outline, Files) could work.

**Comment by ccordoba12** [comment]
I agree with your thought one but I don't understand your thought two.

**Comment by bcolsen** [comment]
> but I don't understand your thought two.

Here's a mockup. The wasted space is in pink

![sypyder side bar](https://user-images.githubusercontent.com/10513354/46548413-3ed62b80-c88c-11e8-91ea-41478c6f40fa.png)

Most editors have a side bar and some have tabs on the side, but most editors(and LibreOffice) also have lots of horizontal space because they only have the side bar and the editor.  Spyder is running out of horizontal space with the console and variable explorer also taking horizontal space.

I think that the side tabs are great for discoverability and the the current tabs on the bottom are not very discoverable (I feel this is true for all the panels in spyder). Rstudio and other IDEs have panel tabs at the top of the section which is more discoverable and more common.

**Comment by ccordoba12** [comment]
> Spyder is running out of horizontal space with the console and variable explorer also taking horizontal space.

So you're suggesting that we use a vertical tabbar, right? Because that's what's proposed here. We could use the wasted space for future plugins (like a Git one).

> Rstudio and other IDEs have panel tabs at the top of the section which is more discoverable and more common

This should be doable in Qt (I guess).

**Comment by bcolsen** [comment]
>So you're suggesting that we use a vertical tabbar, right? Because that's what's proposed here. We could use the wasted space for future plugins (like a Git one).

I'm suggesting to use tabs on top of all the panels including the side bar, because vertical tabs have lots of wasted space on very long narrow panels. 

On the other hand if we think we can fill this space with lots of new interesting widgets then I'm in. We could throw "find in files" over there too
